conda/meta.yaml

```diff
@@ -11,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   noarch: python
-  entry_points:
-    - zppy = zppy.__main__:main
```

Comment:
Entry points no longer need to be defined in conda recipes and removing them removes one more maintenance step we would otherwise have to do.